client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -51,6 +52,27 @@ module.exports = React.createClass( {
 		} );
 	},
 
+	monthlyPrice: function() {
+		const { cost, product_slug: productSlug, currency } = this.props.cartItem,
+			// A hacky way to check if we are in signup, since for some reason cartItem.extra.context
+			// is missing in that case (it seems cartItem.extra is overwritten somewhere between)
+			isInSignup = this.props.cartItem.extra && this.props.cartItem.extra.context !== 'calypstore',
+			isPaidPlan = productSlug === 'value_bundle' || productSlug === 'business-bundle';
```

Comment:
Did you have a look to the [`isPremium()` and `isBusiness()` functions](https://github.com/Automattic/wp-calypso/blob/master/client/lib/products-values/index.js#L66)? They operate on a product but maybe they would work with a cart item (I don't remember exactly).